warp/src/constellation/item.rs

```diff
@@ -30,6 +30,15 @@ pub enum FormatType {
     Mime(mediatype::MediaTypeBuf),
 }
 
+impl From<FormatType> for FileType {
```

Comment:
These types seem identical, why not just have one?

Reply:
They are identical at the current time. `FormatType` will have additional fields later that would be irrelevant to `FileType`, although previously it did have additional fields that were removed.